floss/main.py

```diff
@@ -592,65 +592,58 @@ def create_ida_script_content(sample_file_path, decoded_strings, stack_strings):
         if ds.s != "":
             sanitized_string = sanitize_string_for_script(ds.s)
             if ds.characteristics["location_type"] == LocationType.GLOBAL:
-                main_commands.append("print \"FLOSS: string \\\"%s\\\" at global VA 0x%X\"" % (sanitized_string, ds.va))
+                main_commands.append("print(\"FLOSS: string \\\"%s\\\" at global VA 0x%X\")" % (sanitized_string, ds.va))
                 main_commands.append("AppendComment(%d, \"FLOSS: %s\", True)" % (ds.va, sanitized_string))
             else:
-                main_commands.append("print \"FLOSS: string \\\"%s\\\" decoded at VA 0x%X\"" % (sanitized_string, ds.decoded_at_va))
+                main_commands.append("print(\"FLOSS: string \\\"%s\\\" decoded at VA 0x%X\")" % (sanitized_string, ds.decoded_at_va))
                 main_commands.append("AppendComment(%d, \"FLOSS: %s\")" % (ds.decoded_at_va, sanitized_string))
-    main_commands.append("print \"Imported decoded strings from FLOSS\"")
+    main_commands.append("print(\"Imported decoded strings from FLOSS\")")
 
     ss_len = 0
     for ss in stack_strings:
         if ss.s != "":
             sanitized_string = sanitize_string_for_script(ss.s)
             main_commands.append("AppendLvarComment(%d, %d, \"FLOSS stackstring: %s\", True)" % (ss.fva, ss.frame_offset, sanitized_string))
             ss_len += 1
-    main_commands.append("print \"Imported stackstrings from FLOSS\"")
+    main_commands.append("print(\"Imported stackstrings from FLOSS\")")
 
-    script_content = """from idc import RptCmt, Comment, MakeRptCmt, MakeComm, GetFrame, GetFrameLvarSize, GetMemberComment, SetMemberComment, Refresh
+    script_content = """get_cmt, set_cmt, get_func_attr, get_member_cmt, set_member_cmt
```

Comment:
should these be imported here?

Reply:
It seems it is not needed to import them 🙄